WORKSPACE

```diff
@@ -21,78 +21,143 @@ workspace(name = "vaticle_typedb_driver")
 # Load @vaticle_dependencies #
 ##############################
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
This http_archive should be wrapped in a macro in `dependencies` and called like `csharp` and any other rule:
```
# Load //builder/csharp
load("@vaticle_dependencies//builder/csharp:deps.bzl", dotnet_deps = "deps")
dotnet_deps()
```
Check it out, please.